resources/images/model/images.go

```diff
@@ -341,6 +342,11 @@ func getMetaFromArchive(r *io.Reader) (*images.SoftwareImageMetaArtifactConstruc
 		return nil
 	}
 
+        aReader.ScriptsReadCallback = func(r io.Reader, fi os.FileInfo) error {
```

Comment:
We do not, but this was to simplify the integration in future. There are some tasks for the backend and maybe we will show scripts in UI.